README.md

```diff
@@ -103,7 +103,7 @@ Actions.statusUpdate();
 
 There are a couple of hooks avaiable for each action.
 
-* `preEmit` - Is called before the action emits an event. It receives the arguments from the action invocation.
+* `preEmit` - Is called before the action emits an event. It receives the arguments from the action invocation. If it returns something other than undefined, that will be used as arguments for `shouldEmit` and subsequent emission.
```

Comment:
@krawaller Hi there, I'm trying to grasp this feature... can you clarify `subsequent emission` please?